cpp/plugins/cucim.kit.cuslide/src/cuslide/tiff/ifd.cpp

```diff
@@ -365,13 +366,22 @@ const std::vector<uint64_t>& IFD::image_piece_bytecounts() const
 
 bool IFD::is_compression_supported() const
 {
-    return (compression_ == COMPRESSION_ADOBE_DEFLATE || compression_ == COMPRESSION_JPEG ||
-            compression_ == COMPRESSION_DEFLATE);
+    switch (compression_)
+    {
+    case COMPRESSION_NONE:
+    case COMPRESSION_JPEG:
+    case COMPRESSION_ADOBE_DEFLATE:
+    case COMPRESSION_DEFLATE:
+        return true;
+    default:
+        return false;
```

Comment:
Might be worth indenting these to make this a bit more readable

Reply:
Thanks! Will do.

Reply:
Ah ok. Maybe we can track C++ formatting clean as an issue

Reply:
Yeah, will create an issue for that.

Reply:
https://github.com/rapidsai/cucim/issues/116